requirements/dev-requirements.in

```diff
@@ -16,3 +16,4 @@ modernize
 Fabric==1.14.0
 gnureadline==6.3.8
 pip-tools~=3.0
+recommonmark==0.5.0
```

Comment:
Should `dev-requirements` import all from `docs-requirements`?

Reply:
Good point - I can't think of a good reason to ever pin this particular requirement.  I'll drop that, move it to docs-requirements, and import that here.

Reply:
punting on importing from `docs-requirements` - it also imports from requirements, and I get pip-compile errors about installing from github without `-e`

Reply:
yeah. i thought that may be the case. seems fine to punt for now